lib/stormpath-sdk/resource/base.rb

```diff
@@ -57,6 +57,10 @@ def non_printable_properties
       @non_printable_properties ||= []
       Array.new @non_printable_properties
     end
+
+    def has_status(name: :status, status_list: :default)
```

Comment:
Oh, ok. Makes sense. 

Yeah, please move so it's only used by the resources that have a status property.